resources/uploads/attributes/uploads_metadata.yml

```diff
@@ -0,0 +1,16 @@
+# TODO update the documentation on https://help.lob.com/best-practices/campaign-audience-guide about metadata
+title: Metadata
+type: object
+required:
+  - columns
+properties:
+  columns:
+    type: array
+    description: The list of column name from the csv file which you want associated with each of your mailpiece
+    default: []
+    items:
+      type: string
+default: { columns: [] }
+example:
+  columns: ["recipient_name"]
+description: 'The list of column headers in your file as an array that you want as metadata associated with each mailpiece in the format `{ csv_column_name: value_of_row_for_csv_column_name }`. See our <a href="https://help.lob.com/best-practices/campaign-audience-guide" target="_blank">Campaign Audience Guide</a> for additional details.'
```

Comment:
> that you want as metadata associated with each mailpiece in the format `{ csv_column_name: value_of_row_for_csv_column_name }`

The way that's worded I find a little confusing.  It makes me think as a user I'm supposed to provide the `metadata.columns` values in this format.  Maybe leave that part off and just say "The list of column headers in your file as an array that you want as metadata associated with each mailpiece."  And then let them read more about it in the `Campaign Audience Guide` for more information.

Reply:
Oh gotcha.. Okay. My intention was to let them know how the values from the column name will be used as metadata.
But that's a good point. I can update that part in the other guide 👍🏼